shimming-toolbox/shimmingtoolbox/cli/b0shim.py

```diff
@@ -1461,112 +1461,207 @@ def max_intensity(fname_input, fname_mask, fname_output, verbose):
 
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--anat', 'fname_anat', nargs=1, type=click.Path(exists=True), required=False,
-              help="Target image to shim. Supported formats: .nii, .nii.gz")
 @click.option('-i', '--input', 'fname_input', nargs=1, type=click.Path(exists=True), required=True,
               help="Text file containing the shim coefficients. Supported formats: .txt")
 @click.option('-i2', '--input2', 'fname_input2', nargs=1, type=click.Path(exists=True), required=True,
               help="Text file containing the shim coefficients. Supported formats: .txt")
+@click.option('-o', '--output', 'fname_output', type=click.Path(),
+              default=os.path.join(os.path.abspath(os.curdir), 'shim_coefs.txt'),
+              show_default=True, help="Filename to output shim text file.")
+@click.option('-v', '--verbose', type=click.Choice(['info', 'debug']), default='info',
+              help="Be more verbose")
+def add_shim_coefs(fname_input, fname_input2, fname_output, verbose):
+    """ Combine the shim coefficients from two files into a single file."""
+    # Set logger level
+    set_all_loggers(verbose)
+
+    # Prepare the output
+    create_output_dir(fname_output, is_file=True)
+
+    coefs1 = read_txt_file(fname_input)
+    coefs2 = read_txt_file(fname_input2)
+
+    if coefs1.shape == coefs2.shape:
+        coefs = coefs1 + coefs2
+    else:
+        raise ValueError("The number of shim events and/or the number of channels is not the same in both text files")
+
+    logger.debug(coefs1)
+    logger.debug(coefs2)
+    logger.debug(coefs)
+    write_coefs_to_text_file(coefs, fname_output, 'slicewise')
```

Comment:
Always slice-wise?

Reply:
Yes, this could be either 'slicewise' or 'chronological' since the function has no difference when writing them, the difference is in the ordering of the input data. Since the format needs to be the same, we just rewrite in the same order